packages/strapi-plugin-upload/admin/src/components/CardControl/Wrapper.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  width: 24px;
+  height: 24px;
+  margin-right: 5px;
+  background-color: #ffffff;
+  border: 1px solid #e3e9f3;
+  border-radius: 2px;
+  cursor: pointer;
+  font-size: 11px;
+  color: ${({ color }) => color};
+
+  > svg {
+    margin: auto;
+  }
```

Comment:
Maybe instead of set an attribute to svg, you can use `align-items: center;` to the flex parent. It avoid generic selector.